src/services/homework/hooks/index.js

```diff
@@ -365,7 +394,7 @@ exports.before = () => ({
 exports.after = {
 	all: [],
 	find: [iff(isProvider('external'), [hasViewPermissionAfter, addStats])],
-	get: [iff(isProvider('external'), [hasViewPermissionAfter, addStats])],
+	get: [iff(isProvider('external'), [hasViewPermissionAfter, addStats, addLessonInfo])],
```

Comment:
was this necessary? where do we use this endpoint?

Reply:
we use it on task details page.